src/index.js

```diff
@@ -55,18 +55,29 @@ class Routes {
 
   getLink (Link) {
     const LinkRoutes = props => {
-      const {route, params, ...newProps} = props
+      const {route, params, path, ...newProps} = props
 
       if (route) {
         Object.assign(newProps, this.findByName(route).getLinkProps(params))
       }
 
+      if (path) {
+        const {route: routeObj, params: paramsObj} = this.match(path)
```

Comment:
We need to check `if (routeObj)`. If no route was found, maybe assign `newProps.href`